libdnf/rpm/package_sack.cpp

```diff
@@ -75,6 +77,112 @@ void PackageSack::Impl::make_provides_ready() {
     provides_ready = true;
 }
 
+void PackageSack::Impl::setup_excludes_includes(bool only_main) {
+    considered_uptodate = false;
+
+    auto & main_config = base->get_config();
+
+    auto & disable_excludes = main_config.disable_excludes().get_value();
+    std::set<std::string> disabled(disable_excludes.begin(), disable_excludes.end());
+
+    if (disabled.count("all") != 0) {
+        return;
+    }
+
+    PackageSet repo_includes(base);
+    PackageSet repo_excludes(base);
```

Comment:
This local variable shadows the class-level `repo_excludes`, which is confusing, especially since the class-level `repo_excludes` isn't being set to anything yet, I'm not really sure what's its purpose.

Reply:
I will rename local variables.